charts/kyverno-policies/templates/NOTES.txt

```diff
@@ -0,0 +1,5 @@
+Thank you for installing {{ .Chart.Name }} {{ .Chart.Version }} 😀
+
+We have installed the "{{ .Values.podSecurityStandard }}" profile of Pod Security Standards.
```

Comment:
Strike reference of pod security standards.

Reply:
okay